### PR TITLE
Providers in China, didi and caocao

### PR DESCRIPTION
**Pull request by zhigang1992**

### CODE REVIEW

File: data/createFile.js [hunk 1]
```diff
@@ -36,7 +37,7 @@ module.exports = async (arr, company) => {
     { semi: false, parser: 'babylon' }
   )
 
-  await fs.writeFile(`./data/single/${company}.js`, code, err => {
```
Comment (zhigang1992) [inline data/createFile.js]: This was causing issue when running in data folder instead of root

Comment (SaraVieira) [reply data/createFile.js]: Good thinking !

**Comment by zhigang1992** [comment]
Actually no, 😭 
I have not yet tested them.

You can try search for the text in `formatted_address` right, like Chengdu, Beijing etc.

**Comment by SaraVieira** [comment]
Hey !

That doesn't work in algolia, I just search for the name and I also try to make all the names english, do you mind updting that ? :/

**Comment by zhigang1992** [comment]
Sure thing, on it.

**Comment by SaraVieira** [comment]
Thank you again!

Merged and deployed 🎉

You can see it here: https://isthereuber.in/hangzhou

I also added you to the list of contributors

**Pull request merged**